config/build.ts

```diff
@@ -40,31 +39,50 @@ const defaultApiSets = {
 };
 
 
-(() => {
-    Promise.resolve()
-        .then(processSnippets)
+(async() => {
```

Comment:
That's fine, you do have my permission :-).  As long as the playlist looks good (which actually I just validated, and it does), that's fine.  I just know that Shashwat is currently making some changes to the deploy script, so I wanted to make sure this wouldn't be impacted.  But it looks like it's not.  Don't worry about it.